jenkins_jobs/defaults.yml

```diff
@@ -15,4 +15,34 @@
           timeout: 25
           # Mark the build as failed
           fail: true
-
+    triggers:
+      - github-pull-request:
+          # List of admins who can trigger jobs for PRs opened by unrecognized users.
+          admin-list:
```

Comment:
This list should probably correspond with the list of all (non-emerti) [current GPII Committers.](https://wiki.gpii.net/w/GPII_Technical_Governance#Current_Committers).